src/xo-app/backup/restore/index.js

```diff
@@ -98,13 +101,45 @@ const doImport = ({ backup, sr, start }) => {
   }
 }
 
+class _Collapsible extends Component {
+  render () {
+    const {
+      collapsible,
+      children,
+      ...props
+    } = this.props
+
+    return collapsible
+      ? <Collapse {...props}> {children} </Collapse>
+      : <div> {children} </div>
+  }
+}
+
 class _ModalBody extends Component {
   get value () {
     return this.state
   }
 
+  _srPredicate = createSelector(
+    sr => sr,
+    () => this.state.sr.$pool,
+    (sr, pool) => isSrWritable(sr) && sr.$pool === pool
+  )
+
+  _getSelectedValue = vdi => {
+    const value = this.state.vdiSr && this.state.vdiSr[vdi]
+    if (value && value.$pool !== this.state.sr.$pool
+    ) {
+      delete this.state.vdiSr[vdi]
+      return undefined
```

Comment:
I need to delete the selected sr from the state because it is not accessible in this case.